gulpfile.js

```diff
@@ -130,14 +129,32 @@ const serverconfig = {
     publicPath: true,
   },
 };
-// Load up the function that will be used to start a webpack dev server
-// This does NOT start the server, that requires the gulp task below.
-const webpackdevserver = require('./tools/tasks/webpack-dev-server')(wpconfig, serverconfig);
+
+// Scope a holder to our webpack dev server here
+let wpds = null;
+// Destructure the args off of webpack-dev-server library
+const {
+  startWebpackDevServer, // Load up the function that will be used to start a webpack dev server
+  refreshWebpackDevServer, // Refresh an instance of a webpack dev server
+} = require('./tools/tasks/webpack-dev-server');
+
+/**
+ * Starts up the Webpack Dev Server, lots going on here:
+ * 1. Instantiate a webpack dev server with all config it needs
+ * 2. Once instantiated, send it the callback that every gulp function requires
+ * 3. Return all this to an actual instance of the server we can play with in other tasks
+ */
+gulp.task('webpack:server', (cb) => {
+  wpds = startWebpackDevServer(wpconfig, serverconfig)(cb);
+});
 
 /**
- * Starts up the Webpack Dev Server with our config from aove
+ * Refresh an active instance of webpack dev server
  */
-gulp.task('webpack:server', webpackdevserver);
+gulp.task('webpack:refresh-server', (cb) => {
+  // Note: wpds lives in scope above this.
```

Comment:
What's a wpd?

Reply:
@grayside, note the comment mentioning the parent scope. Var is instantiated on [line 134](https://github.com/phase2/pattern-lab-starter/pull/134/files/f85af2f98fad2537f9854ffab0dd30cd150aafac#diff-b9e12334e9eafd8341a6107dd98510c9R134).

Reply:
Shorthand for webpack dev server I believe. Set on 134 and altered.

Reply:
Noting here that with more time and elegance, I'd store the wpds as a singleton. Basically, it would look like:

    // instantiation
    webpackdevserver.start(wpconfig, serverconfig)(cb)

    // Use
    webpackdevserver.reload();

Dammit, @grayside, now I really want to do this.

Reply:
I generally avoid acronym variables because reviewing diffs is harder, but I grok now.